spec/helpers/helpers.rb

```diff
@@ -0,0 +1,10 @@
+module Helpers
+  def login_user(name: 'demouser', email: 'demouser@viget.com')
+    user = create(:user, username: name, email: email)
+    visit new_sessions_path
+    fill_in "Username:", with: user.username
+    fill_in "Password:", with: user.password
+    click_button 'Sign In'
+    return user
```

Comment:
just for your knowledge here, using `tap` is a neat way to avoid an explicit return:
```ruby
create(:user).tap do |user|
  visit ...
  fill_in ..., with: user.username
  click_button ...
end
```
[Documentation](https://apidock.com/ruby/Object/tap) is a bit cryptic, but `tap` lets you make use of the object that you call it on in the supplied block, and then returns that object.